app/src/main/kotlin/com/wire/android/ui/home/conversations/model/UIMessage.kt

```diff
@@ -42,7 +42,8 @@ data class MessageHeader(
     val messageId: String,
     val userId: UserId? = null,
     val connectionState: ConnectionState?,
-    val isDeleted: Boolean
+    val isDeleted: Boolean,
+    val isUnavailable: Boolean
```

Comment:
It's a little confusing, because it gives the impression that the message is unavailable, but in fact it's about the user, maybe we could rename it?

Reply:
good point